app/src/main/java/app/revanced/integrations/settingsmenu/ReVancedSettingsFragment.java

```diff
@@ -139,6 +145,7 @@ public void onCreate(Bundle bundle) {
             enableDisablePreferences();
 
             externalDownloaderPreferenceScreen = (PreferenceScreen) getPreferenceScreen().findPreference("external_downloader");
+            whitelistingPreferenceScreen = (PreferenceScreen) getPreferenceScreen().findPreference("whitelisting");
```

Comment:
The cause of `Whitelist Setting is not shown`  (Not declared in `revanced_pref.xml`)

Reply:
Thanks, I will check later